CHANGES_NEXT_RELEASE

```diff
@@ -3,3 +3,4 @@
 - Hardening: Several changes in argument passing in mongoBackend library to add 'const' in references to objects that are not altered by the function
 - Fix: bug in parseArg lib that may cause problem printing the error message for wrong CLI usage (#2926)
 - Fix: bug in variable substitution of custom notifications that limited the size of the payload of a custom notification to 1024 bytes (new limit: 8MB)
+- Fix: "request payload too large" (>1MB) as Bad Input alarm (WARN log level)
```

Comment:
I have found that there was two places in which this is checked in rest.cpp. In one of them, we were already using alarmMgr.badInput(). Thus, I see this as a kind of error fixing, so mentioning it in CNR.

NTC (just informative)